model/__init__.py

```diff
@@ -0,0 +1,3 @@
+from model.amity import Amity
+from model.person import *
+from model.room import *
```

Comment:
this was some experiment i was trying and forgot to delete them. Will delete them immediately